### DISCUSSION TITLE
Sensor foam pad thickness

### DISCUSSION THREAD

**Discussion by polygonfuture** [srozum/film_camera_tester#19] [Q&A] [answered]
Hey Serhiy,

What thickness is the (what looks like neoprene) foam pads for the photo sensor strain relief?  Eyeballing it looks maybe around 4mm?

![image](https://user-images.githubusercontent.com/12026545/227113336-4f5854bb-cae8-4398-bc6f-26537caada22.png)

**Comment by srozum** [comment, marked as answer]
It's a foam. It used to press photodiode to the front frame. Use as thick as you need.
